bin/ch/WScriptJsrt.cpp

```diff
@@ -2022,6 +2022,16 @@ HRESULT WScriptJsrt::ModuleMessage::Call(LPCSTR fileName)
                     }
                 }
             }
+            else
```

Comment:
Did you look into whether it's possible (or makes sense) to add a test case to `bin/NativeTests/JsRTApiTest.cpp`? I haven't dug around in that file so I'm not sure. If we can and it's not too much work then this probably belongs over there.

Otherwise, I think it would be fine for this PR to just make the change to `SourceTextModuleRecord.cpp` and we can worry about the test later.

Reply:
JsRTApiTest doesn't currently test JsModuleEvaluation so it would mean adding a new test to it - that aside I don't really know how that file works so for now I have updated to not have a test and merely revert the unintentional change.